README.rst

```diff
@@ -0,0 +1,85 @@
+======
+mtools
+======
+
+|PyPI version| |Build Status| |Python 27|
+
+**mtools** is a collection of helper scripts to parse, filter, and visualize
+MongoDB log files (``mongod``, ``mongos``). mtools also includes ``mlaunch``, a
+utility to quickly set up complex MongoDB test environments on a local machine.
+
+.. figure:: ./mtools.png
+   :alt: mtools box
+
+What's in the box?
+------------------
+
+The following tools are in the mtools collection:
+
+`mlogfilter <https://github.com/rueckstiess/mtools/wiki/mlogfilter>`__ slices
```

Comment:
In MarkDown the tool description appears on a line below the tool name. With GitHub RST rendering the description follows immediately after the tool name, which doesn't flow well. Can you add delimiters or newlines?

Reply:
Yep. A definition list should work nicely here.